docker/docker-compose.yml

```diff
@@ -43,11 +43,10 @@ services:
     ports:
       - 33807:33807
     healthcheck:
-      test: ['CMD', 'mysqladmin', 'ping', '-h127.0.0.1', '-P33577']
+      test: ['CMD', 'mysqladmin', 'ping', '-h127.0.0.1', '-P33807']
```

Comment:
This was incorrect making that the container is always marked unhealthy. Health state is needed for `planetscale_proxy`.